1_Mechanics/02_GMGuide.md

```diff
@@ -1,82 +1,263 @@
+story rather than against them. 
+
+You should be an impartial force in the world, controlling both the bad and
+the good without judgement, anger, or aggression. The best sessions feel like
+everyone is working together to complete an objective, even the GM, rather
+than it feeling like a battle or competition between the Players and the Game
+Master. This is a delicate balance, and GM's are encouraged to ask for their
+Player's feedback on how they can improve, and it is also a GM's
+responsibility to provide feedback to the Players at the table when necessary
+to help everyone enjoy their experience at the table. 
+
+### Game Master Resources
+
+GM's should have several decks of cards available to them during a session so
+as to easily represent multiple groups of Minions and Dealers that the
+Players will fight during the course of an Adventure. At minimum, a GM must
+maintain two separate decks - one for enemy combatants and one for allies of
+the Players. If a GM only has one deck available, they should split the deck
+into two 26 card decks and re-shuffle regularly. 
+
+As the Game Master, you do not have to draw a single deck to empty. Unless a
+Dealer you control has an individual deck assigned to them, and they are in
+the active scene for long enough, the GM never invokes Lower Hand by needing
+to reshuffle. If an NPC were to need a mechanical reason to show them being
+exhausted, the same Status Condition can be added to that NPC as if they had
+a level of fatigue.  
+
+### Managing Player Resources
+
+As the Game Master, you can design Adventures at your Table to test different
+aspects of the Players and characters. Each Player holds a single deck of 52
+cards, with two Fate Cards reserved at the beginning of play. Each character
+has statistics that include total Health Points (HP), Fate Points (FP), and
+Armor Points (AP). 
+
+You can think of a Player's deck of cards like their character's stamina. The
+Adventurers should get progressively more tired throughout the day. This is
+accomplished by having the Players make a series of Checks, which is their
+way of interacting with the world. At lower levels, it is OK for Players not
+to approach the re-draw limit of running out of cards between Rests. However,
+as players progress and level up, the Powers they choose and the Encounters
+they face may test them closer to running out of cards between Rests. 
+
+### Resting
+
+HP and FP are also balanced by Rests. It is expected and recommended that a
+party will take 1-2 Quick Rests between Full Rests during each day of in-game
+time, depending on the Encounters they face. While Resting, the Players will
+be regaining their Health and Fate Points, expending Rest Cards along the
+way. This should be kept in mind when planning Encounters that deal damage,
+since Players will want to take a Quick Rest if their HP or FP drops below a
+threshold they feel comfortable with. 
```

Comment:
I think you could revise systematically in quasi-second person, as this entire file is directed at GMs. This will cut down on the word count. 

```suggestion
be regaining their Health and Fate Points, and expending Rest Cards. 
Keep this in mind when planning Encounters that deal damage,
since Players will want to take a Quick Rest if their HP or FP drop too low. 
```